lib/pages/company_site/product_creation_page.rb

```diff
@@ -0,0 +1,372 @@
+    button(:save, css: '.aui-admin-button_submit')
+
+    # Удаление и выход из редактирования
+    # Открывается попап удаления/выхода с одинаковыми классами у обеих кнопок, но разным текстом
+    button(:quit, css: '.product .page-info__edit-product-control-buttons_quit')
+    button(:delete, css: '.product .page-info__edit-product-control-buttons_delete')
+    span(:edit_popup, css: '.product .page-info-popup__block')
+    button(:edit_yes, css: '.page-info-popup__block [type="submit"]')
+    button(:edit_no, css: '.page-info-popup__block .page-info-popup__link')
+
+    # Модерация
+    #   Кнопки на форме
+    button(:accept, css: '.aui-admin-moderation-button_accept')
+    button(:decline, css: '.aui-admin-moderation-button_decline')
+    button(:postpone, css: '.aui-admin-moderation-button_postpone')
+    #   Попап модерации
+    span(:moderation_popup, css: '.moderation-popup__block')
+    textarea(:cause_input, css: '.moderation-popup__block .moderation-popup__textarea')
+    select(:cause_select, css: '.moderation-popup__block .aui-admin-select__input')
+    button(:moderation_submit, css: '.moderation-popup__block .aui-admin-button')
+    #   Уведомление
+    span(:moderation_message, css: 'div.page-info__notice')
+
+    # Батарейка
+    span(:thermometer_degree, css: '.aui-admin-battery__percent')
+    link(:thermometer_degree_popup_link, css: '.aui-admin-battery__link')
+    span(:thermometer_degree_popup, css: '.aui-admin-popup')
+
+    # ОПИСАНИЕ МЕТОДОВ
+    #
+    # Основной метод последовательного заполнения атрибутов, переданых в переменной options из спека
+    # Если атрибут есть в переменной options, то заполняется; если атрибута нет, то пропускается
+    # def fill_attributes(options)
+    #   self.name_input = options[:name] if options[:name]
+    #   self.article_input = options[:article] if options[:article]
+    #   self.exists_select = options[:exists] if options[:exists]
+
+    #   fill_price(options) if options[:price]
+    #   fill_price_range(options) if options[:price_from]
+    #   fill_discount_price(options) if options[:price_old]
+    #   fill_wholesale_price(options) if options[:wholesale_price]
+
+    #   self.short_description = options[:short_description] if options[:short_description]
+    #   self.description = options[:description] if options[:description]
+    #   self.tag_title_input = options[:tag_title] if options[:tag_title]
+
+    #   fill_rubric(options) if options[:rubric]
+    #   fill_priority_in_rubric(options) if options[:priority_lvl4]
+    #   fill_company_traits(options) if options[:company_traits]
+    #   fill_portal_traits(options) if options[:portal_traits]
+
+    #   load_image(options) if options[:path_to_image]
+    #   fill_group(options) if options[:group]
+    #   load_doc(options) if options[:path_to_doc]
+    #   fill_soputka(options) if options[:soputka]
+    #   self.min_qty_input = options[:min_qty] if options[:min_qty]
+
+    #   save
+    # end
+
+    # 
```

Comment:
[Correctable] Layout/EmptyComment: Source code comment is empty.<br>[Correctable] Layout/TrailingWhitespace: Trailing whitespace detected.